src/menu.tsx

```diff
@@ -1,15 +1,15 @@
 import * as React from 'react';
 import classnames from 'classnames';
-import {assignInlineVars} from '@vanilla-extract/dynamic';
 import {ESC, TAB} from './utils/key-codes';
 import {cancelEvent, getPrefixedDataAttributes} from './utils/dom';
 import Overlay from './overlay';
 import * as styles from './menu.css';
-import {sprinkles} from './sprinkles.css';
+import {Portal, useWindowSize} from '.';
 
 import type {DataAttributes} from './utils/types';
 
-const MAX_HEIGHT_DEFAULT = 416;
```

Comment:
I think the Menu should have a max height to avoid edge cases where there are to much items in the menu to fit in the screen (in small screens) or showing a huge list (in large screens). In the select, for example, we have a max height